docker/falco/Dockerfile

```
@@ -22,6 +22,7 @@ RUN apt-get update \
 	ca-certificates \
 	curl \
 	dkms \
+	flex \
```

```suggestion
	flex \
	bison \
```

Also `bison`.

We put them in the `driverkit` builder ([ref](https://github.com/falcosecurity/driverkit/blob/master/build/builder.Dockerfile#L26)) but forgot to add them here.

See [here](https://elixir.bootlin.com/linux/latest/source/Documentation/process/changes.rst)

---

Hi @leodido ,

Thanks for the review. Good point. Would you like me to update the PR for the bison package?